discord/guild.py

```diff
@@ -4292,3 +4333,45 @@ async def create_automod_rule(
         )
 
         return AutoModRule(data=data, guild=self, state=self._state)
+
+    @property
+    def invites_paused_until(self) -> Optional[datetime.datetime]:
+        """:class:`datetime.datetime`: When invites get enabled again.
+
+        .. versionadded:: 2.4
+        """
+        if not self._incidents_data:
+            return None
+
+        return utils.parse_time(self._incidents_data.get("invites_disabled_until"))
+
+    @property
+    def dms_paused_until(self) -> Optional[datetime.datetime]:
+        """:class:`datetime.datetime`: When direct messages get enabled again.
```

Comment:
```suggestion
        """Optional[:class:`datetime.datetime`]: If DMs are paused, returns when DMs will get enabled in UTC,
        otherwise returns None.
```